### ISSUE TITLE
hactoberfest

### ISSUE THREAD

**Issue by vkassingh** [vkassingh/The-Layman-Machine-Learning-dictionary#4]

**Comment by ronika-das** [comment]
Hello. Can you describe the issue please?

**Comment by Dhruv-VINT** [comment]
ML algos stuff is all good and fine. But how about adding a ML-prerequisites folder too, which will include all the essential tutorials for python libraries (Pandas, Seaborn, Scikitlearn, etc) required to build a solid foundation before jumping on ML concepts ?

**Comment by AniruthA** [comment]
Maybe we could also get in some code for some simple examples using the algorithms. I think that would really help people out, because I learn best when I'm hands-on. I could add some if you'd like!

**Comment by vkassingh** [comment]
Yes why not!
You are free to make contributions and learn. So go ahead!

Make quality contribution!

Have a Good day!
Vikas Singh

**Comment by AniruthA** [comment]
Ok @vkassingh ! I finished two examples (Logistic and Linear Regression), written from scratch. By the way, could you add the hacktoberfest topic to your repo. I would like for this to count towards my PR count.

**Comment by vkassingh** [comment]
Sure I will add both the hacktober fest labels today!

Have a great day ahead
Vikas Singh

**Comment by AniruthA** [comment]
Thanks a lot @vkassingh .  I hope you have. Great day as well!

**Comment by AniruthA** [comment]
I added the Q Learning (Reinforcement Learning) example!

**Comment by AniruthA** [comment]
I also added the Minimax example.

**Comment by vkassingh** [comment]
> I also added the Minimax example.

well you can create a seperate folder of examples with seperate topics and remember It should be beginner freindly.

**Comment by vkassingh** [comment]
> ML algos stuff is all good and fine. But how about adding a ML-prerequisites folder too, which will include all the essential tutorials for python libraries (Pandas, Seaborn, Scikitlearn, etc) required to build a solid foundation before jumping on ML concepts ?

Yes good idea, you are free to make contribtutions here! go ahead

**Comment by Suvodeep-Das** [comment]
Full Name: Suvodeep Das
GitHub Profile Link: https://github.com/Suvodeep-Das
Objective: I would like to provide some codes on some basic machine learning algorithms like logistic regression, Decision tree and random forest.
Hacktoberfest Participant.

I would like to work on this issue.

**Comment by vkassingh** [comment]
Sure! Don't forget to add examples and applications of the concept.

**Comment by aliikrhnn** [comment]
first try
